### ISSUE TITLE
What `ShuffleNet_1X_g3_br_0.25_373.hdf5` file is trained from?

### ISSUE THREAD

**Issue by GewelsJI** [scheckmedia/keras-shufflenet#7]

**Comment by GewelsJI** [comment]
how should I use it in the binary classification case?

**Comment by scheckmedia** [comment]
It's trained on imagenet with 1000 classes. If you want to use it for binary classification you have to load the model drop the last dense layer and add a dense layer with one unit and sigmoid as activation function.

**Comment by GewelsJI** [comment]
firstly, thx your nice answer

but I have another is :
what should I set hyper-parameter (such as optimizer should I choose etc.)

and:
Should I remove the TOP-layer and just add `dense` layer after the last CONV-layer? or use $1\times1$ filter to replace FC-layer?
@scheckmedia

**Comment by scheckmedia** [comment]
Just set include_top to false while you initialize the model. As next step you can add additional dense layers where the last layer contains one unit. 

For the question about hyper parameters I've no answer because there is no setup which works always right. It depends on your problem, dataset a.s.o. The implementation I use for shufflenet are the proposed settings from the paper.

**Comment by GewelsJI** [comment]
thx for your nice answer!

**Comment by scheckmedia** [comment]
You are welcome. I'll close this issue now.
